internal/command/meta_backend.go

```diff
@@ -1021,7 +1021,9 @@ func (m *Meta) backend_C_r_S_changed(c *configs.Backend, cHash int, sMgr *clista
 		if c.Type == "cloud" {
 			output = fmt.Sprintf(outputBackendMigrateChangeCloud, s.Backend.Type)
 		}
-		m.Ui.Output(strings.TrimSpace(output))
+		m.Ui.Output(m.Colorize().Color(fmt.Sprintf(
+			"[reset]%s\n",
+			strings.TrimSpace(output))))
```

Comment:
Not completely sure what the reason for these two changes is, but I assume it's fixing up some errant double newlines. Is that correct?

Reply:
Yeah; there was no gap between some of the lines you see in the image above, and it looked sorta bad to me; we can iterate more later but I figured I made as well take some artistic liberty and clean it up.